src/views/Accessibility/AccessibiltyRequest/Documents/New/index.tsx

```diff
@@ -150,7 +183,91 @@ const New = () => {
                       ariaDescribedBy=""
                     />
                   </Label>
+                  {selectedFile && (
+                    <FieldGroup
+                      scrollElement="documentType.commonType"
+                      error={!!flatErrors['documentType.commonType']}
+                    >
+                      <fieldset className="usa-fieldset margin-top-4">
+                        <legend className="usa-label margin-bottom-1">
+                          What type of document are you uploading?
+                        </legend>
 
+                        <FieldErrorMsg>
+                          {flatErrors['documentType.commonType']}
+                        </FieldErrorMsg>
+                        {([
```

Comment:
You might be able to do some magic here to use the strings from the type and remove the "Other" option, but this is ok for now.

Reply:
I tried to do that, but couldn't figure it out, plus they have to be in a different order than alphabetical, anyway I think this is good for now.